From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi@gmx.de>
Date: Sun, 8 May 2022 22:53:34 +0200
Subject: [PATCH] Remove an include of `err.h`

`err.h` appears to be BSD-specific, and we don't support it. Luckily,
imgcat compiles just fine even when removing it.
---
 src/imgcat.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/imgcat.c b/src/imgcat.c
index 26a73d7f359cc9ce30b2ba02a2b1f46617afada5..89fc05d19ae85786d115fee8eb16869e02b51513 100644
--- a/src/imgcat.c
+++ b/src/imgcat.c
@@ -22,7 +22,6 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <err.h>
 #include <limits.h>
 
 #include <getopt.h>
